plumgrid_playbooks/plumgrid_common.yml

```diff
@@ -20,9 +20,13 @@
     - plumgrid_common
     - create_ifcs
     - plumgrid_director
+  any_errors_fatal: yes
   vars:
+    pg_package_name: plumgrid-lxc
     pg_service_name: plumgrid
-  any_errors_fatal: yes
+    plumgrid_data_dir: "/var/lib/libvirt/filesystems/plumgrid-data"
```

Comment:
since director is going with plumgrid-lxc in either case, thats why had to keep it static. In this scenario there is no other way around.
ok, i'll change dictionary name.